### PR TITLE
Freshly Baked Docs 

### PR DESCRIPTION
**Pull request by RustyRaptor**
All done with the front page. This revision should do for now. I personally think the 'info' folder is no longer necessary and makes it harder to maintain the docs. If we move to a sort of documentation framework that updates them automatically that would make more sense.

### CODE REVIEW

**Comment by marios8543** [comment]
@carmelo12341 take a look

File: README.md [hunk 1]
```diff
@@ -1,73 +1,98 @@
 # Kamina
+
+
+[](logo/kamina_logo.svg)
+
+[//]: # (This and the logo line are comments for Ziad to uncomment when he edits because)
+[//]: # (he likes to see the pretty logo in pycharm when he edits the docs uwu )
```
Comment (ghost) [inline README.md:6]: OwO

File: README.md [hunk 2]
```diff
@@ -1,73 +1,98 @@
...
+
+
+
+*Kamina* will be split into 3 separate components that will communicate with each other. 
+
+- ***Kamina Community***
+- ***Kamina User***
+- ***Kamina Search***
+
+The APIs for each component will be implemnted in Flask and the front end of each will be implemented using VueJS.
```
Comment (ghost) [inline README.md]: Typo there, "implemnted"

**Comment by ghost** [comment]
Also, there are a lot of empty lines. Too much imo.

**Comment by RustyRaptor** [comment]
Alright will fix it up today.

**Comment by RustyRaptor** [comment]
Alright it's fixed. I also ran it through grammarly and Yoast SEO. Yoast gave a suggestion about sentence length. Some of the sentences need to be shortened so it's easier for everyone to read so I will make an issue for that.

**Comment by ghost** [comment]
Yeah, its much better now, Thanks!

**Pull request merged**